app/src/main/java/com/berniesanders/connect/MainActivity.java

```diff
@@ -32,8 +48,21 @@ protected void onCreate(Bundle savedInstanceState) {
         mText.setText(mContext.getPackageName());
 
         mActionAlertsModel.getActionAlerts().subscribe(
-                alerts -> Timber.d("alerts: " + alerts),
-                error -> {},
-                () -> {});
+                alerts -> mAlertListView.setAdapter(new ActionAlertAdapter(mContext, alerts)),
```

Comment:
curiously, why is there an injected context here? the MainActivity is a context, and we should be able to access the  ApplicationContext from this activity